app/src/main/java/com/sun/dailynote/data/model/Note.java

```diff
@@ -0,0 +1,115 @@
+package com.sun.dailynote.data.model;
+
+import com.sun.dailynote.data.database.Database;
+
+import java.util.Date;
+import java.util.Objects;
+
+public class Note {
+    private int mId;
+    private Date mDateCreate;
+    private String mContent;
+    private String mName;
+    private String mImage;
+    private Date mTimeStart;
+    private boolean mStatus;
+    private Date mTimeEnd;
+    private int mCategory;
+
+    public Note(int mId,
+                Date mDateCreate,
+                String mContent,
+                String mName,
+                String mImage,
+                Date mTimeStart,
+                boolean mStatus,
+                Date mTimeEnd,
+                int mCategory) {
+        this.mId = mId;
+        this.mDateCreate = mDateCreate;
+        this.mContent = mContent;
+        this.mName = mName;
+        this.mImage = mImage;
+        this.mTimeStart = mTimeStart;
+        this.mStatus = mStatus;
+        this.mTimeEnd = mTimeEnd;
+        this.mCategory = mCategory;
+    }
+
+    public int getmCategory() {
+        return mCategory;
+    }
+
+    public void setmCategory(int mCategory) {
```

Comment:
thiết đặt code style, java, code generate, cho field prefix là m để khi sinh code nó không bị như thế này
```suggestion
    public void setCategory(int category) {
```